lib/src/styles/dimensions.dart

```diff
@@ -0,0 +1,8 @@
+abstract class Dimension {
+  static const double shadowAppBar = 20.0;
```

Comment:
Also lets get a common practice to get all these `Config` files into a package `common` what do you think ?

Reply:
It's a good point. Alredy changed the package name from 'styles' to 'utils'